<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="EventDeliveryFailedException.xml" source-language="en-US" target-language="ko-KR">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-15c36f0" tool-company="Microsoft" />
      <xliffext:skl_file_name xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">02cd5861-7ce2-4a82-b358-31f8435a0ac548b889544946a903dd75b269a8b9f2adb8827aff.skl</xliffext:skl_file_name>
      <xliffext:version xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">1.2</xliffext:version>
      <xliffext:ms.openlocfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">48b889544946a903dd75b269a8b9f2adb8827aff</xliffext:ms.openlocfilehash>
      <xliffext:ms.sourcegitcommit xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">d31dc2ede16f6f7bc64e90d9f897ff54c4e3869b</xliffext:ms.sourcegitcommit>
      <xliffext:ms.lasthandoff xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">04/03/2018</xliffext:ms.lasthandoff>
      <xliffext:moniker_ids xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">netframework-4.5.1,netframework-4.5.2,netframework-4.5,netframework-4.6.1,netframework-4.6.2,netframework-4.6,netframework-4.7.1,netframework-4.7</xliffext:moniker_ids>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>The exception that is thrown when an event that is raised from the host cannot be delivered to the workflow instance.</source>
          <target state="translated">호스트에서 발생한 이벤트를 워크플로 인스턴스에 전달할 수 없는 경우 throw되는 예외입니다.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>Typically the event is raised from an <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.ExternalDataExchangeService" /&gt;</ph> on a workflow instance.</source>
          <target state="translated">일반적으로 이벤트는 워크플로 인스턴스의 <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.ExternalDataExchangeService" /&gt;</ph>에서 발생합니다.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>This class cannot be inherited.</source>
          <target state="translated">이 클래스는 상속될 수 없습니다.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>The following string is added to the event log when this exception is thrown: Event "{1}" on interface type "{0}" for instance id "{2}" cannot be delivered.</source>
          <target state="translated">이 예외가 throw되면 다음 문자열이 이벤트 로그에 추가됩니다． 인스턴스 ID "{2}"에 대한 인터페이스 유형 "{0}"의 "{1}" 이벤트를 전달할 수 없습니다.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>When using a state machine workflow, you might get an exception with the message "Queue '{0}' is not enabled."</source>
          <target state="translated">상태 시스템 워크플로를 사용하면 "'{0}'을(를) 사용할 수 없습니다."라는 메시지와 함께 예외가 발생할 수 있습니다.</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>This happens when the current state of the state machine does not know how to handle a specific event.</source>
          <target state="translated">이 예외는 상태 시스템의 현재 상태에서 특정 이벤트를 처리하는 방법을 모르고 있기 때문입니다.</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve" extradata="MT" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>For example, when some state other than the current state contains the <ph id="ph1">&lt;xref:System.Workflow.Activities.EventDrivenActivity&gt;</ph> that contains the <ph id="ph2">&lt;xref:System.Workflow.Activities.HandleExternalEventActivity&gt;</ph> that is represented by the queue '{0}'.</source>
          <target state="translated">예를 들어 현재 상태 이외의 일부 상태에 '{0}' 큐에서 나타내는 <ph id="ph1">&lt;xref:System.Workflow.Activities.EventDrivenActivity&gt;</ph>를 포함하는 <ph id="ph2">&lt;xref:System.Workflow.Activities.HandleExternalEventActivity&gt;</ph>가 있습니다.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" uid="T:System.Workflow.Activities.EventDeliveryFailedException">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> class.</source>
          <target state="translated"><ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String)">
          <source>An error message that explains the reason for the exception.</source>
          <target state="translated">예외에 대한 이유를 설명하는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> class by using the error message that explains the reason for the exception.</source>
          <target state="translated">예외에 대한 이유를 설명하는 오류 메시지를 사용하여 <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다.</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String,System.Exception)">
          <source>An error message that explains the reason for the exception.</source>
          <target state="translated">예외에 대한 이유를 설명하는 오류 메시지입니다.</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String,System.Exception)">
          <source>The <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph> that caused this exception.</source>
          <target state="translated">이 예외를 발생시킨 <ph id="ph1">&lt;see cref="T:System.Exception" /&gt;</ph>입니다.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve" uid="M:System.Workflow.Activities.EventDeliveryFailedException.#ctor(System.String,System.Exception)">
          <source>Initializes a new instance of the <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph> class by using the error message that explains the reason for the exception and the <ph id="ph2">&lt;see cref="T:System.Exception" /&gt;</ph> that caused this exception.</source>
          <target state="translated">예외에 대한 이유 및 이 예외를 발생시킨 <ph id="ph1">&lt;see cref="T:System.Workflow.Activities.EventDeliveryFailedException" /&gt;</ph>을 설명하는 오류 메시지를 사용하여 <ph id="ph2">&lt;see cref="T:System.Exception" /&gt;</ph> 클래스의 새 인스턴스를 초기화합니다</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>